### PR TITLE
Closes #68. Email verification

### LINKED ISSUE #68: Email verification

### PR DESCRIPTION
**Pull request by gallyamb**

### CODE REVIEW

**Review by LeMarck — APPROVED** [review]

**Review by Grehhov — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ const hbsHelpers = require('./utils/hbs-helpers');
 const error = require('./middlewares/error');
 const passport = require('./configs/passport.js');
 const upload = require('./configs/multer');
-require('./configs/recaptcha')();
+require('./configs/recaptcha');
+require('./configs/nev');
```
Comment (gallyamb) [inline index.js]: Remove

File: index.js [hunk 2]
```diff
@@ -72,5 +73,5 @@ connectToDb()
                 console.info(`Open http://localhost:${port}/ to view service`);
             }
         });
-    })
+    }, err => console.error(err))
```
Comment (gallyamb) [inline index.js]: Remove

File: models/user/temp-user.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            type: Date,
+            expires: 24 * 60 * 60,
+            default: Date.now
+        }
+    }
+);
+
+const tempUserSchema = new Schema(tempUserSchemaObject);
+
+tempUserSchema.pre('save', function (next) {
```
Comment (gallyamb) [inline models/user/temp-user.js:22]: What if the persistent user will want to change password?

Comment (gallyamb) [reply models/user/temp-user.js:22]: It's a separate issue "Allow user to change password"

**Pull request merged**